tests/functional/test_check.py

```diff
@@ -292,3 +294,23 @@ def test_check_include_work_dir_pkg(script):
     )
     assert matches_expected_lines(result.stdout, expected_lines)
     assert result.returncode == 1
+
+
+@pytest.mark.xfail(
+    reason='Not yet implemented: https://github.com/pypa/pip/pull/8633',
+    strict=True,
```

Comment:
Could you please add `raises=` arg here indicating what sort of exception is currently expected to happen?

I guess it's some kind of a subprocess error, right?